File: Jr/2024/Sushi.java
// Owen Bélanger-Love

import java.util.Scanner;

public class Sushi {
    
    public static void main(String[] args) {
        
        // Create input Scanner
        Scanner input = new Scanner(System.in);

        // Get amount of red sushi
        int red = input.nextInt();

        // Get amount of green sushi
        int green = input.nextInt();

        // Get amount of blue sushi
        int blue = input.nextInt();

        // Calculate price
        int price = 3 * red + 4 * green + 5 * blue;

        // Print the total price
        System.out.println(price);

        // Close Scanner
        input.close();
    }
}

File: Jr/2024/Sushi.c
// Owen Bélanger-Love
#include <stdio.h>

int main() {
    int red, green, blue;

    // Get the amount of red sushi plates
    scanf("%d", &red);
    
    // Get the amount of green sushi plates
    scanf("%d", &green);

    // Get the amount of blue sushi plates
    scanf("%d", &blue);

    // Calculate the total cost
    int total = 3 * red + 4 * green + 5 * blue;

    // Print total price
    printf("%d", total);

    return 0;
}